src/snippets.ts

```diff
@@ -0,0 +1,139 @@
+import { SignerProvider } from '@alephium/web3'
+import { InvariantInstance, TokenFaucetInstance } from '../artifacts/ts'
+import { LiquidityScale, MinSqrtPrice, PercentageScale } from './consts'
+import {
+  getPool,
+  initPool,
+  initPositionWithLiquidity,
+  initSwap,
+  initTokensXY,
+  withdrawTokens
+} from './testUtils'
+import { balanceOf, deployInvariant } from './utils'
+import { PrivateKeyWallet } from '@alephium/web3-wallet'
+
+type TokenInstance = TokenFaucetInstance
+
+// 6% fee
+const fee = 6n * 10n ** (PercentageScale - 3n)
+const tickSpacing = 10n
+
+export const initBasicFeeTickSpacing = (): [bigint, bigint] => {
```

Comment:
You dont initialize anything, it would be more convinient to name it `getBasicFeeTickSpacing`